src/borg/helpers.py

```diff
@@ -2162,13 +2162,13 @@ def heuristic_lz4(self, compr_args, chunk):
         cdata = lz4.compress(data)
         data_len = len(data)
         cdata_len = len(cdata)
-        if cdata_len < data_len:
+        if cdata_len < 0.97*data_len:
```

Comment:
let's see if that triggers the pep8 checker. usually, spaces around operators are desired.